store/storeIface.go

```diff
@@ -104,5 +106,9 @@ type Store interface {
 
 	SaveIndexerBaseHeight(height uint64) error
 
+	LoadLastSettlementBlockTime() (uint64, error)
```

Comment:
why u need in store? it can `Manager` field
it synced from SL on boot
* FN follow the hub states
* sequencer sets on submission